Student/src/main/java/org/example/servlets/AddStudentServlet.java

```diff
@@ -0,0 +1,30 @@
+package org.example.servlets;
+
+import org.example.entities.Student;
+import org.example.model.Model;
+
+import javax.servlet.ServletException;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import java.io.IOException;
+
+public class AddStudentServlet extends HttpServlet {
+    @Override
+    protected void doGet(HttpServletRequest req, HttpServletResponse resp) throws ServletException, IOException {
+        getServletContext().getRequestDispatcher("/WEB-INF/add.jsp").forward(req, resp);
+    }
+
+    @Override
+    protected void doPost(HttpServletRequest req, HttpServletResponse resp) throws ServletException, IOException {
+        String name = req.getParameter("name");
+        String surname = req.getParameter("surname");
+        Integer age = Integer.valueOf(req.getParameter("age"));
+        Student student = new Student(name, surname, age);
+        Model studentModel = Model.getStudentModel();
+        studentModel.add(student);
+
+        req.setAttribute("student", name);
```

Comment:
I think, it's better to name  request attribute `studentName` instead of `student` as you put only name, rather then the whole object